ci-operator/step-registry/hypershift/hive/create/hostedcluster/hypershift-hive-create-hostedcluster-commands.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+AWS_GUEST_INFRA_CREDENTIALS_FILE="/etc/hypershift-ci-jobs-awscreds/credentials"
```

Comment:
Can you add information about how to use this in the workflow documentation?

Reply:
I actually plan to remove it (copied from hypershift team workflow), since user should always provide a `cluster_profile`